go.mod

```diff
@@ -1,34 +1,36 @@
 module github.com/factrylabs/factry-historian-datasource.git
 
-go 1.22.1
+go 1.23
```

Comment:
Also update README to mention this. (Or remove the version specifier in the readme)